daemons/fenced/fenced_remote.c

```diff
@@ -1727,6 +1727,27 @@ request_peer_fencing(remote_fencing_op_t *op, peer_device_info_t *peer)
     crm_trace("Action %.8s targeting %s for %s is %s",
               op->id, op->target, op->client_name,
               stonith_op_state_str(op->state));
+
+    if ((op->phase == st_phase_on) && (op->devices != NULL)) {
+        /* We are in the "on" phase of a remapped topology reboot. If this
+         * device has pcmk_reboot_action="off", skip it.
+         *
+         * We can't check device properties at this point because we haven't
+         * chosen a peer for this stage yet. Instead, we check the local node's
+         * configuration for the device, which should be the same across all
+         * nodes.
+         */
+        device = op->devices->data;
+        if (pcmk__str_eq(fenced_device_reboot_action(device), "off",
+                         pcmk__str_none)) {
+            crm_info("Skipping 'on' using %s in remapped reboot targeting %s "
+                     "since it has pcmk_reboot_action='off'",
+                     device, op->target);
+            advance_topology_device_in_level(op, device, NULL);
```

Comment:
This is of course correct but something like
'Skipping turning back 'on' in remapped reboot targeting node1 as arbitrary-reboot-device is explicitly configured to stay 'off' (pcmk_reboot_action='off').'
might make it easier for the less involved human.

Reply:
How about:

Not turning node1 back on using device1 because the device is configured to stay off (pcmk_reboot_action='off')

Reply:
And:

Not turning node1 back on using device1 because the agent doesn't support 'on'

Reply:
Cool - short and simple - had left the reference to remapping in but we probably don't need it and it is simpler to read without